### PR TITLE
Update automated-device-enrollment-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added clarity to mention that  all features mentioned above are not supported in Apple setup Assistant authentication methods.

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4020

### CODE REVIEW

File: memdocs/intune/enrollment/automated-device-enrollment-authentication.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Use the Intune Company Portal app as the authentication method if you want to:
  - Automatically install the Company Portal app during enrollment. If your company uses the Volume Purchase Program (VPP), you can automatically install Company Portal app during enrollment without user Apple IDs. 
  - You want to lock the device until the Company Portal app installs.
 
-These features aren't supported in Apple Setup Assistant authentication methods. 
+All The features mentioned above aren't supported in Apple Setup Assistant authentication methods.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/automated-device-enrollment-authentication.md:58]:
```suggestion
All of the features mentioned in Option 1 aren't supported in Apple Setup Assistant authentication methods. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: memdocs/intune/enrollment/automated-device-enrollment-authentication.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Use the Intune Company Portal app as the authentication method if you want to:
  - Automatically install the Company Portal app during enrollment. If your company uses the Volume Purchase Program (VPP), you can automatically install Company Portal app during enrollment without user Apple IDs. 
  - You want to lock the device until the Company Portal app installs.
 
-These features aren't supported in Apple Setup Assistant authentication methods. 
+All of the features mentioned in Option 1 aren't supported in Apple Setup Assistant authentication methods.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/automated-device-enrollment-authentication.md:58]:
```suggestion
None of the features mentioned in Option 1 are supported in Apple Setup Assistant authentication methods. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**